.github/README.md

```diff
@@ -107,7 +111,7 @@ To build locally:
     poetry run make html
     ```
 
-    If using setuptools:
+    If using pip:
```

Comment:
I guess `make html` is true for any python environment as long as the virtual env is activated. The above poetry one is to use its own virtual env. But it's fine to make the change here.

Reply:
Yeah, I was a bit unsure about what to write here. Perhaps "if not using Poetry" or "if using a different/any other Python virtual environment"?

Reply:
Nah, I guess the current change is fine since the whole README emphasize the difference between pip and poetry. Might be better to keep the same here. It just reminds me of the strange issue on M1 recently.